mkdocs.yml

```diff
@@ -10,6 +10,7 @@ nav:
   - Concepts:
     - Sides: 'concepts/sides.md'
     - Resources: 'concepts/resources.md'
+    - Datapacks: 'concepts/datapacks.md'
```

Comment:
The name should probably just be "Data" to match "Resources"

Reply:
I put it to data now. Could potentially put datagen in that file? or an intro to it. Did not rename the title inside the file though.